install_linux.sh

```diff
@@ -64,6 +65,30 @@ fi
 echo -e "\n\n===================================================="
 echo "Unpacking ${download_zip} ..."
 unzip -u "${download_zip}" -d "${download_path}"
+
+if [ "${SKIP_VERIFY_CHECKSUM}" != "y" ] && [ "${SKIP_VERIFY_CHECKSUM}" != "yes" ]; then
+  echo "Verifying checksum with sha256sum..."
+  set +e
+
+  cd "${download_path}"
+  curl -s -L -o "${checksum_file}" "https://github.com/terraform-linters/tflint/releases/download/${version}/checksums.txt"
+  sha256sum --ignore-missing -c "${checksum_file}" >/dev/null 2>&1
+
+  retVal=$?
+  if [ $retVal -eq 127 ]; then
+    echo "  verify: sha256sum not found, ignore verifying checksum and continue installation!"
+  else
+    if [ $retVal -ne 0 ]; then
+      echo "  verify: sha256sum verification failed with error code $retVal, exit installation!"
+      rm -rf "${download_path}"
```

Comment:
If needed, move the existing cleanup logic to a `trap` than runs on exit rather than duplicating it